native/android/src/main/java/com/nozbe/watermelondb/Database.kt

```diff
@@ -8,20 +8,21 @@ import android.database.sqlite.SQLiteDatabase
 import android.database.sqlite.SQLiteQuery
 import java.io.File
 
-class Database(private val name: String, private val context: Context) {
+class Database(private val name: String, private val context: Context, private val openFlags: Int = SQLiteDatabase.CREATE_IF_NECESSARY or SQLiteDatabase.ENABLE_WRITE_AHEAD_LOGGING) {
```

Comment:
keep the code clean. Split that line (if you are using AS to edit this stuff just hit ⌥⌘L)

Reply:
meh, would be nice if prettier could reformat it as I went in VSCode, since AS is borderline unusable on M1, and ktlint didn't complain

Reply:
@rozPierog fixed

Reply:
We can think about integrating some prettier plugin for KTlint. I'll look into this for Nozbe internal repo to see if we can easliy reuse same rules as ktlint uses'

> ktlint didn't complain

That's worrisome, and indicates that it might have broke